packages/block-library/src/column/block.json

```diff
@@ -12,6 +12,9 @@
 		"width": {
 			"type": "string"
 		},
+		"allowedBlocks": {
+			"type": "array"
+    	},
```

Comment:
Would be good to get the white space correct here.

Reply:
Fixed in the new PR: https://github.com/WordPress/gutenberg/pull/35342 👍